openedx_learning/apps/authoring/units/api.py

```diff
@@ -30,7 +30,12 @@
 
 
 def create_unit(
-    learning_package_id: int, key: str, created: datetime, created_by: int | None
+    learning_package_id: int,
+    key: str,
+    created: datetime,
+    created_by: int | None,
+    *,
+    can_stand_alone: bool = True,
```

Comment:
Please add this to the list of Args in each of the function docstrings too.

Reply:
Done